packages/wasm-dpp/src/data_contract/data_contract.rs

```diff
@@ -227,12 +235,27 @@ impl DataContractWasm {
     #[wasm_bindgen(js_name=toObject)]
     pub fn to_object(&self) -> Result<JsValue, JsValue> {
         let serializer = serde_wasm_bindgen::Serializer::json_compatible();
-        with_js_error!(self.0.serialize(&serializer))
+        let object = with_js_error!(self.0.serialize(&serializer))?;
+
+        js_sys::Reflect::set(
+            &object,
+            &Into::<JsValue>::into("$id".to_owned()),
+            &Into::<JsValue>::into(Buffer::from_bytes(&self.0.id.buffer)),
+        )
+        .expect("target is an object");
```

Comment:
I think it's better to convert that to an error with a simple message, like "oops, something went wrong". The reason is that expect will bloat js runtime by quite a bit

Reply:
hmm, I used it because there is nothing that can go wrong and it's just an `unwrap` with documentation string